wandb/board/ui/src/App.js

```diff
@@ -40,13 +39,11 @@ class App extends Component {
       <div className={this.props.fullScreen ? 'fullScreen' : ''}>
         <AutoReload setFlash={this.props.setFlash} />
         <Nav user={this.props.user} history={this.props.history} />
-        <div className="main" style={{padding: '20px'}}>
-          {this.props.error || this.state.error ? (
-            <ErrorPage history={this.props.history} error={this.props.error} />
-          ) : (
-            this.props.children
-          )}
-        </div>
+        <ErrorPage {...this.props}>
```

Comment:
@vanpelt this is the reference that I was mentioning https://reactjs.org/blog/2017/07/26/error-handling-in-react-16.html

Reply:
You’re using componentDidCatch?